src/plots/cartesian/set_convert.js

```diff
@@ -360,7 +361,6 @@ module.exports = function setConvert(ax) {
             Lib.notifier(
                 'Something went wrong with axis scaling',
                 'long');
-            ax._gd._replotting = false;
             throw new Error('axis scaling');
```

Comment:
If we think the error is fatal to the plot then yes, it's enough. But if there would be a use case for catching the error and continuing, then the plot will be in an odd state without resetting `gd._replotting`. I suppose at the very least we can move it to `fullLayout` but we should really figure out a better solution than the fragile `_replotting` flag to prevent infinite recursion.